### ISSUE TITLE
[IntelliJ] Get NotFoundHttpErrorStatus exception when refresh HDInsight Node in Azure Explorer

### ISSUE THREAD

**Issue by t-rufang** [microsoft/azure-tools-for-java#3642]
Repro Steps:
1. Sign in Azure account and select subscription `HDInsight VS Tool Team`
2. Refresh HDInsight node in Azure Explorer. We will find the error mark in the bottom right in IntelliJ.

```
Error getting cluster configurations with NEW HDInsight API. /subscriptions/7f31cba8-b597-4129-b158-8f21a7395bd0/resourceGroups/1314/providers/Microsoft.HDInsight/clusters/sparknotebook4

com.microsoft.azure.hdinsight.sdk.cluster.HDInsightNewAPI.HDInsightNewApiUnavailableException: com.microsoft.azure.hdinsight.sdk.common.errorresponse.NotFoundHttpErrorStatus: {"error":{"code":"ResourceNotFound","message":"The Resource 'Microsoft.HDInsight/clusters/sparknotebook4' under resource group '1314' was not found."}}
	at com.microsoft.azure.hdinsight.sdk.cluster.HDInsightNewAPI.ClusterOperationNewAPIImpl.lambda$isProbeGetConfigurationSucceed$1(ClusterOperationNewAPIImpl.java:127)
	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4.onError(OperatorOnErrorResumeNextViaFunction.java:140)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onError(OnSubscribeMap.java:88)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onError(OnSubscribeMap.java:88)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.reportError(OperatorMerge.java:266)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.checkTerminate(OperatorMerge.java:818)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.emitLoop(OperatorMerge.java:579)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.emit(OperatorMerge.java:568)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onError(OperatorMerge.java:855)
	at rx.observers.Subscribers$5.onError(Subscribers.java:230)
	at rx.internal.operators.OperatorDoAfterTerminate$1.onError(OperatorDoAfterTerminate.java:56)
	at rx.internal.operators.OnSubscribeThrow.call(OnSubscribeThrow.java:44)
	at rx.internal.operators.OnSubscribeThrow.call(OnSubscribeThrow.java:28)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OnSubscribeUsing.call(OnSubscribeUsing.java:94)
	at rx.internal.operators.OnSubscribeUsing.call(OnSubscribeUsing.java:32)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.onNext(OperatorMerge.java:248)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.onNext(OperatorMerge.java:148)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:77)
	at rx.internal.producers.SingleDelayedProducer.emit(SingleDelayedProducer.java:102)
	at rx.internal.producers.SingleDelayedProducer.setValue(SingleDelayedProducer.java:85)
	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:48)
	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:33)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
	at rx.Observable.unsafeSubscribe(Observable.java:10327)
	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
	at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:230)
	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.microsoft.azure.hdinsight.sdk.common.errorresponse.NotFoundHttpErrorStatus: {"error":{"code":"ResourceNotFound","message":"The Resource 'Microsoft.HDInsight/clusters/sparknotebook4' under resource group '1314' was not found."}}
	at com.microsoft.azure.hdinsight.sdk.common.HttpObservable.classifyHttpError(HttpObservable.java:299)
	at com.microsoft.azure.hdinsight.sdk.common.HttpObservable.lambda$toStringOnlyOkResponse$1(HttpObservable.java:326)
	at rx.internal.operators.OnSubscribeUsing.call(OnSubscribeUsing.java:66)
	... 32 more
```

**Comment by t-rufang** [comment]
Fixed in #3638

**Comment by v-lanjli** [comment]
verified and fixed in https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1258/Azure/processDownloadRequest/artifacts/azure-toolkit-for-intellij-2019.2.develop.1258.09-29-2019.zip
And closed this issue.
